randovania/games/am2r/json_data/Distribution Center.txt

```diff
@@ -1455,21 +1450,40 @@ Extra - map_name: rm_a5b05
   > Tunnel to Spiky Spider Trial Reward Room
       Any of the following:
           All of the following:
-              Morph Ball and Mid-Air Morph (Advanced)
+              # Suitless Method
+              Morph Ball
+              Any of the following:
+                  # Get past crumble blocks
+                  Mid-Air Morph (Advanced)
+                  Power Grip and Movement (Intermediate)
               Any of the following:
+                  # To climb the room
                   All of the following:
                       Walljump (Intermediate)
-                      Power Grip or Mid-Air Morph (Expert)
+                      Power Grip or Mid-Air Morph (Advanced)
                   All of the following:
                       # IBJ
                       Infinite Bomb Jumping (Advanced) and Can Use Bombs
-          Gravity Suit and Can Use Spider Ball
+          All of the following:
+              # Gravity Suit Method
+              Gravity Suit
+              Any of the following:
+                  # Get past crumble blocks
+                  Can Use Spider Ball
+                  All of the following:
+                      Hi-Jump Boots and Spring Ball and Movement (Intermediate)
+                      Any of the following:
+                          # To climb the room
+                          Hijump Tunnel Climb
+                          Hi-Jump Boots and Mid-Air Morph (Intermediate)
```

Comment:
Comments still seem off in this section. I'd probably put spider in a seperate section with "Spider method", and then have the rest follow a similar structure as gravityless